### PR TITLE
nuttx: update licenses to Apache

### PR DESCRIPTION
**Pull request by jerpelea**
## Summary
several authors have submitted the SGA or ICLA and we can migrate the licenses to Apache.
Alan Carvalho de Assis
David S. Alessio
David Sidrane
Gregory Nutt
Sebastien Lorquet

## Impact
NONE

## Testing
NONE

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]
LGTM

File: boards/arm/imxrt/imxrt1060-evk/src/imxrt_boot.c [hunk 1]
```diff
@@ -64,9 +64,10 @@ void imxrt_ocram_initialize(void)
   regval = getreg32(IMXRT_IOMUXC_GPR_GPR16);
   putreg32(regval | GPR_GPR16_FLEXRAM_BANK_CFG_SELF, IMXRT_IOMUXC_GPR_GPR16);
 
-  for (src = (uint32_t *)(LOCATE_IN_SRC(g_boot_data.start) + g_boot_data.size),
+  for (src = (uint32_t *)(LOCATE_IN_SRC(g_boot_data.start) +
+       g_boot_data.size),
        dest = (uint32_t *)(g_boot_data.start + g_boot_data.size);
-       dest < (uint32_t *) &_etext;)
+       dest < (uint32_t *) &_etext);
```
Comment (xiaoxiang781216) [inline boards/arm/imxrt/imxrt1060-evk/src/imxrt_boot.c:70]: change to:
```
dest < (uint32_t *) &_etext; )
```

Comment (jerpelea) [reply boards/arm/imxrt/imxrt1060-evk/src/imxrt_boot.c]: done

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by jerpelea** [comment]
@xiaoxiang781216 
boards/arm/imxrt/imxrt1060-evk/src/imxrt_boot.c:70:36: error: Space precedes right parenthesis

**Comment by xiaoxiang781216** [comment]
> @xiaoxiang781216
> boards/arm/imxrt/imxrt1060-evk/src/imxrt_boot.c:70:36: error: Space precedes right parenthesis
> remains

I will take a look after merging.

**Comment by jerpelea** [comment]
are you merging it or should I ?

**Comment by xiaoxiang781216** [comment]
Done.

**Pull request merged**